README.md

```diff
@@ -1 +1,83 @@
-# todo-list
\ No newline at end of file
+# TODO LIST
+
+> The purpose of this project is to build a todo list app in order to showcase our mastery over object oriented design in javascript. 
+  
+## :hammer: Built With
+
+- HTML5
+- Javascript ES6
+- Webpack
+- Bulma
+
+## Live Demo
+
+[Todo App](https://livedemo.com)
+
+
+## Getting Started
+
+To get a local copy up and running follow these simple steps.
+
+### Prerequisites
+  
+  You need to have git installed on your local system.
+  You can clone this project with the command `$ git clone https://www.github.com/mariobarrioss/todo-list.git`.
+  For this project you also require NPM installed on your machine.
+  
+### Setup
+  
+  No Further setup is required beyond cloning the repo on your local machine.
+  
+### Install
+  
+  To install the dependencies for the application run the following commands in your terminal
+  
+  `$ npm install`
+  
+### Usage
+  
+  To build the appplication the following command needs to be run in the terminal
+  
+  `$ npm run buiid`
+  
+### Run tests
+  
+  There are no tests for this project.
+  
+### Deployment
+  
+  This section will be updated later.

```

Comment:
## Add  the live link to where the page is deployed
- [ ] Be sure, you are able to showcase your work by ensuring the page is hosted and its link provided here in the README for a more solid and descriptive README.md ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/general/descriptive_readme.png) from [Pathwright](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/66243643/)).